crates/types/src/contract_address.rs

```diff
@@ -11,6 +11,12 @@ impl ContractAddress {
     pub fn new(felt: Felt) -> DevnetResult<Self> {
         Ok(Self(PatriciaKey::new(felt)?))
     }
+
+    /// Constructs a zero address
+    pub fn zero() -> Self {
+        // using unwrap because we are sure it works for 0x0
+        Self::new(Felt::from_prefixed_hex_str("0x0").unwrap()).unwrap()
```

Comment:
Felt implemented from u128, that way you are going go avoid the first unwrap.
Something like this `Self::new(Felt::from(0)).unwrap()`

Reply:
Addressed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/76/commits/744e49d02d4331571033d65b7dc6c03a8e97c68d